### ISSUE TITLE
Incorrect coverage when running multiple tests

### ISSUE THREAD

**Issue by twifty** [twifty/atom-phpunit-integrator#21]
Running all files/suites results in less coverage than running the single default suite. This discrepancy can be as much as 40%. Obviously there is a problem with how the reports are merged.

**Comment by twifty** [comment]
Upon further investigation I have discovered the `<file>` entries in the clover.xml reports are inconsistent between tests. Take the following PHP code:

```php
class Hello
{
    public function sayHello()
    {
        return 'Hello';
    }
}
```
When running the `HelloTest` suite which actually covers the `Hello` class, those single line opening braces are excluded from the `<line>` list. Running any other test suite, the coverage report will include those single braces as a `<line>` with the `type="stmt"` attribute.

Well, these are not executable lines, so they are not statements. I'm not sure if this is a bug within `php-code-coverage` or not. Either way it is miss matched line count between two file reports. Those extra lines will always be marked as uncovered throwing the merged result way out of whack.

**Comment by twifty** [comment]
Fixed in 0.0.23

One problem that will not fix is the statement counts will be wrong for other files when running a single test. Since the coverage for such files will always be 0%, it should not cause a problem. It is only noticeable within the coverage panel where the covered and total statements for each file are displayed.
